tests/unit/models/QuotasServiceTest.php

```
@@ -76,6 +76,14 @@ public function testEditQuota()
 
         $quotaData['name'] = 'UpdateQuotaName';
 
+        // Setup $_POST data for language settings
+        $_POST['QuotaLanguageSetting'] = [];
+        foreach ($quota->languagesettings as $language => $languageSetting) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$languageSetting'. (undefined)

(UnusedLocalVariable)